.eslintrc

```diff
@@ -53,10 +53,16 @@
         "no-self-compare": "error",
         "no-setter-return": "error",
         "object-curly-spacing": ["error", "always"],
-
         // TS rules
         "@typescript-eslint/prefer-optional-chain": "error",
-        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-unused-vars": [
```

Comment:
Please remove these changes. We don't want to have any unused variables in our code.)